app/src/main/java/net/gsantner/markor/frontend/textview/TextViewUtils.java

```diff
@@ -352,7 +345,11 @@ public static void showSelection(final TextView text, final int start, final int
         text.requestRectangleOnScreen(region);
     }
 
-    public static void setSelectionAndShow(final EditText edit, final int... sel) {
+    public static void showSelection(final TextView text) {
+        showSelection(text, text.getSelectionStart(), text.getSelectionEnd());
+    }
+
+    public static void setSelectionAndShow(final EditText edit, boolean setSelection, final int... sel) {
```

Comment:
We have `showSelection` to show the selection and `setSelectionAndShow` to set selection and show. Having a boolean flag here to setSelection in a function called `setSelectionAndShow` is poor design imo. Just call the right function

Reply:
Okay, I have made some improvements for this.